### ISSUE TITLE
[ItemsAdder] Unhandled exception occurred in onPacketSending(PacketEvent) for ItemsAdder

### ISSUE THREAD

**Issue by Neast1337** [PluginBugs/Issues-ItemsAdder#3715]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### Discord tag (optional)

neast1407

### What happened?

an error in the console, which may have created an out-of-memory server crash when the server has 15 GB of free space.

### Steps to reproduce the issue

the error is random. it is displayed without doing anything special on the server

### Server version

[14:46:17 INFO]: This server is running Paper version git-Paper-550 (MC: 1.19.4) (Implementing API version 1.19.4-R0.1-SNAPSHOT) (Git: 483368e on ver/1.19.4)
You are running the latest version
Previous version: git-Paper-379 (MC: 1.18.2)

### ItemsAdder Version

[14:46:29 INFO]: ItemsAdder version 3.6.4-release-3
[14:46:29 INFO]: Customized Minecraft without client mods!
[14:46:29 INFO]: Website: devs.beer
[14:46:29 INFO]: Author: LoneDev

### ProtocolLib Version

[14:46:44 INFO]: ProtocolLib version 5.2.0-SNAPSHOT-679
[14:46:44 INFO]: Provides read/write access to the Minecraft protocol.
[14:46:44 INFO]: Authors: dmulloy2 and comphenix

### LoneLibs Version

[15:50:52 INFO]: LoneLibs version 1.0.45
[15:50:52 INFO]: Collection of libs used by my plugins
[15:50:52 INFO]: Authors: LoneDev and others

### Error (optional)

```shell
[02:12:49] [Server thread/ERROR]: [ItemsAdder] Unhandled exception occurred in onPacketSending(PacketEvent) for ItemsAdder
java.lang.UnsupportedOperationException: com.google.gson.JsonSyntaxException: Expected id to be a string, was an array ([775...57])
	at lonelibs.net.kyori.adventure.platform.bukkit.MinecraftComponentSerializer.serialize(MinecraftComponentSerializer.java:257) ~[LoneLibs.jar:?]
	at dev.lone.LoneLibs.chat.Comp.componentToNms(Comp.java:180) ~[LoneLibs.jar:?]
	at ia.m.hG.onPacketSending(SourceFile:186) ~[ItemsAdder_3.6.4-release-3.jar:?]
	at com.comphenix.protocol.injector.SortedPacketListenerList.invokeSendingListener(SortedPacketListenerList.java:219) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.SortedPacketListenerList.invokeUnpackedPacketSending(SortedPacketListenerList.java:204) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketSending(SortedPacketListenerList.java:149) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketSending(SortedPacketListenerList.java:139) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.PacketFilterManager.postPacketToListeners(PacketFilterManager.java:553) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.PacketFilterManager.invokePacketSending(PacketFilterManager.java:526) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.netty.manager.NetworkManagerInjector.onPacketSending(NetworkManagerInjector.java:100) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.netty.channel.NettyChannelInjector.processOutbound(NettyChannelInjector.java:553) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.netty.channel.NettyChannelInjector$2.doProxyRunnable(NettyChannelInjector.java:456) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.proxyRunnable(NettyEventLoopProxy.java:43) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.execute(NettyEventLoopProxy.java:252) ~[ProtocolLib.jar:?]
	at net.minecraft.network.Connection.sendPacket(Connection.java:442) ~[?:?]
	at net.minecraft.network.Connection.send(Connection.java:385) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.send(ServerGamePacketListenerImpl.java:2160) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.send(ServerGamePacketListenerImpl.java:2146) ~[?:?]
	at net.minecraft.server.level.ServerPlayer.openMenu(ServerPlayer.java:1548) ~[?:?]
	at net.minecraft.world.item.trading.Merchant.openTradingScreen(Merchant.java:41) ~[?:?]
	at net.minecraft.world.entity.npc.Villager.startTrading(Villager.java:345) ~[?:?]
	at net.minecraft.world.entity.npc.Villager.mobInteract(Villager.java:323) ~[?:?]
	at net.minecraft.world.entity.Mob.interact(Mob.java:1321) ~[?:?]
	at net.minecraft.world.entity.player.Player.interactOn(Player.java:1128) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl$5.performInteraction(ServerGamePacketListenerImpl.java:2836) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl$5.a(ServerGamePacketListenerImpl.java:2856) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket$InteractionAction.dispatch(ServerboundInteractPacket.java:131) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.dispatch(ServerboundInteractPacket.java:80) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleInteract(ServerGamePacketListenerImpl.java:2797) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:67) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:12) ~[?:?]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:51) ~[?:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1342) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1319) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1312) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1290) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1178) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper-1.19.4.jar:git-Paper-550]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: com.google.gson.JsonSyntaxException: Expected id to be a string, was an array ([775...57])
	at net.minecraft.util.GsonHelper.convertToString(GsonHelper.java:85) ~[?:?]
	at net.minecraft.util.GsonHelper.getAsString(GsonHelper.java:91) ~[?:?]
	at net.minecraft.network.chat.HoverEvent$EntityTooltipInfo.create(HoverEvent.java:177) ~[?:?]
	at net.minecraft.network.chat.HoverEvent$Action.deserialize(HoverEvent.java:136) ~[?:?]
	at net.minecraft.network.chat.HoverEvent.deserialize(HoverEvent.java:80) ~[?:?]
	at net.minecraft.network.chat.Style$Serializer.getHoverEvent(Style.java:405) ~[?:?]
	at net.minecraft.network.chat.Style$Serializer.deserialize(Style.java:377) ~[?:?]
	at net.minecraft.network.chat.Style$Serializer.deserialize(Style.java:364) ~[?:?]
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:76) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1214) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1319) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1290) ~[gson-2.10.jar:?]
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:179) ~[gson-2.10.jar:?]
	at net.minecraft.network.chat.Component$Serializer.deserialize(Component.java:361) ~[?:?]
	at net.minecraft.network.chat.Component$Serializer.deserialize(Component.java:222) ~[?:?]
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:76) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1214) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1319) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1261) ~[gson-2.10.jar:?]
	at net.minecraft.network.chat.Component$Serializer.fromJson(Component.java:536) ~[?:?]
	at lonelibs.net.kyori.adventure.platform.bukkit.MinecraftComponentSerializer.serialize(MinecraftComponentSerializer.java:253) ~[LoneLibs.jar:?]
	... 44 more
[02:12:49] [Server thread/ERROR]: Parameters: 
  net.minecraft.network.protocol.game.PacketPlayOutOpenWindow@663608a7[
    a=7
    b=net.minecraft.world.inventory.Containers@2c641718
    c=translation{key='entity.minecraft.villager.cartographer', args=[]}[style={hoverEvent=HoverEvent{action=<action show_entity>, value='net.minecraft.network.chat.ChatHoverable$b@5968dd02'},insertion=2e372720-5e29-412b-95ab-39f0c3b6b713}]
  ]

[02:13:13] [Server thread/ERROR]: [ItemsAdder] Unhandled exception occurred in onPacketSending(PacketEvent) for ItemsAdder
java.lang.UnsupportedOperationException: com.google.gson.JsonSyntaxException: Expected id to be a string, was an array ([775...57])
	at lonelibs.net.kyori.adventure.platform.bukkit.MinecraftComponentSerializer.serialize(MinecraftComponentSerializer.java:257) ~[LoneLibs.jar:?]
	at dev.lone.LoneLibs.chat.Comp.componentToNms(Comp.java:180) ~[LoneLibs.jar:?]
	at ia.m.hG.onPacketSending(SourceFile:186) ~[ItemsAdder_3.6.4-release-3.jar:?]
	at com.comphenix.protocol.injector.SortedPacketListenerList.invokeSendingListener(SortedPacketListenerList.java:219) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.SortedPacketListenerList.invokeUnpackedPacketSending(SortedPacketListenerList.java:204) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketSending(SortedPacketListenerList.java:149) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketSending(SortedPacketListenerList.java:139) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.PacketFilterManager.postPacketToListeners(PacketFilterManager.java:553) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.PacketFilterManager.invokePacketSending(PacketFilterManager.java:526) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.netty.manager.NetworkManagerInjector.onPacketSending(NetworkManagerInjector.java:100) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.netty.channel.NettyChannelInjector.processOutbound(NettyChannelInjector.java:553) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.netty.channel.NettyChannelInjector$2.doProxyRunnable(NettyChannelInjector.java:456) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.proxyRunnable(NettyEventLoopProxy.java:43) ~[ProtocolLib.jar:?]
	at com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.execute(NettyEventLoopProxy.java:252) ~[ProtocolLib.jar:?]
	at net.minecraft.network.Connection.sendPacket(Connection.java:442) ~[?:?]
	at net.minecraft.network.Connection.send(Connection.java:385) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.send(ServerGamePacketListenerImpl.java:2160) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.send(ServerGamePacketListenerImpl.java:2146) ~[?:?]
	at net.minecraft.server.level.ServerPlayer.openMenu(ServerPlayer.java:1548) ~[?:?]
	at net.minecraft.world.item.trading.Merchant.openTradingScreen(Merchant.java:41) ~[?:?]
	at net.minecraft.world.entity.npc.Villager.startTrading(Villager.java:345) ~[?:?]
	at net.minecraft.world.entity.npc.Villager.mobInteract(Villager.java:323) ~[?:?]
	at net.minecraft.world.entity.Mob.interact(Mob.java:1321) ~[?:?]
	at net.minecraft.world.entity.player.Player.interactOn(Player.java:1128) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl$5.performInteraction(ServerGamePacketListenerImpl.java:2836) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl$5.a(ServerGamePacketListenerImpl.java:2856) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket$InteractionAction.dispatch(ServerboundInteractPacket.java:131) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.dispatch(ServerboundInteractPacket.java:80) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleInteract(ServerGamePacketListenerImpl.java:2797) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:67) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:12) ~[?:?]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:51) ~[?:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1342) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1319) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1312) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1290) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1178) ~[paper-1.19.4.jar:git-Paper-550]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper-1.19.4.jar:git-Paper-550]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: com.google.gson.JsonSyntaxException: Expected id to be a string, was an array ([775...57])
	at net.minecraft.util.GsonHelper.convertToString(GsonHelper.java:85) ~[?:?]
	at net.minecraft.util.GsonHelper.getAsString(GsonHelper.java:91) ~[?:?]
	at net.minecraft.network.chat.HoverEvent$EntityTooltipInfo.create(HoverEvent.java:177) ~[?:?]
	at net.minecraft.network.chat.HoverEvent$Action.deserialize(HoverEvent.java:136) ~[?:?]
	at net.minecraft.network.chat.HoverEvent.deserialize(HoverEvent.java:80) ~[?:?]
	at net.minecraft.network.chat.Style$Serializer.getHoverEvent(Style.java:405) ~[?:?]
	at net.minecraft.network.chat.Style$Serializer.deserialize(Style.java:377) ~[?:?]
	at net.minecraft.network.chat.Style$Serializer.deserialize(Style.java:364) ~[?:?]
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:76) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1214) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1319) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1290) ~[gson-2.10.jar:?]
	at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:179) ~[gson-2.10.jar:?]
	at net.minecraft.network.chat.Component$Serializer.deserialize(Component.java:361) ~[?:?]
	at net.minecraft.network.chat.Component$Serializer.deserialize(Component.java:222) ~[?:?]
	at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:76) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1214) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1319) ~[gson-2.10.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1261) ~[gson-2.10.jar:?]
	at net.minecraft.network.chat.Component$Serializer.fromJson(Component.java:536) ~[?:?]
	at lonelibs.net.kyori.adventure.platform.bukkit.MinecraftComponentSerializer.serialize(MinecraftComponentSerializer.java:253) ~[LoneLibs.jar:?]
	... 44 more
[02:13:13] [Server thread/ERROR]: Parameters: 
  net.minecraft.network.protocol.game.PacketPlayOutOpenWindow@7b6ec556[
    a=12
    b=net.minecraft.world.inventory.Containers@2c641718
    c=translation{key='entity.minecraft.villager.cartographer', args=[]}[style={hoverEvent=HoverEvent{action=<action show_entity>, value='net.minecraft.network.chat.ChatHoverable$b@5968dd02'},insertion=2e372720-5e29-412b-95ab-39f0c3b6b713}]
  ]
```

**Comment by Andre601** [comment]
You posted ProtocolLib for the LoneLibs version. Please provide the actual version and also your latest.log file from the Server.

**Comment by Andre601** [comment]
On another note, please mention what client version you join with. And finally is this most likely a duplicate of #3685

**Comment by Neast1337** [comment]
the server is currently in 1.19.4
I connect to the server in 1.20.1 with the Viaversion plugin.
I modified my post because I was mistaken, I have everything c

**Comment by LoneDev6** [comment]
Update LoneLibs

**Comment by Andre601** [comment]
Also, as a general rule: Always test with the same client and server versions.
ViaVersion can and will break stuff.

**Comment by Neast1337** [comment]
it creates the same error again 
Even with the lonelibs update

[16:27:18 ERROR]: [ItemsAdder] Unhandled exception occurred in onPacketSending(PacketEvent) for ItemsAdder
java.lang.UnsupportedOperationException: com.google.gson.JsonSyntaxException: Expected id to be a string, was an array ([140...94])
        at lonelibs.net.kyori.adventure.platform.bukkit.MinecraftComponentSerializer.serialize(MinecraftComponentSerializer.java:257) ~[LoneLibs.jar:?]
        at dev.lone.LoneLibs.chat.Comp.componentToNms(Comp.java:180) ~[LoneLibs.jar:?]
        at ia.m.hG.onPacketSending(SourceFile:186) ~[ItemsAdder_3.6.4-release-3.jar:?]
        at com.comphenix.protocol.injector.SortedPacketListenerList.invokeSendingListener(SortedPacketListenerList.java:219) ~[ProtocolLib (2).jar:?]
        at com.comphenix.protocol.injector.SortedPacketListenerList.invokeUnpackedPacketSending(SortedPacketListenerList.java:204) ~[ProtocolLib (2).jar:?]
        at com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketSending(SortedPacketListenerList.java:149) ~[ProtocolLib (2).jar:?]
        at com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketSending(SortedPacketListenerList.java:139) ~[ProtocolLib (2).jar:?]
        at com.comphenix.protocol.injector.PacketFilterManager.postPacketToListeners(PacketFilterManager.java:555) ~[ProtocolLib (2).jar:?]
        at com.comphenix.protocol.injector.PacketFilterManager.invokePacketSending(PacketFilterManager.java:528) ~[ProtocolLib (2).jar:?]
        at com.comphenix.protocol.injector.netty.manager.NetworkManagerInjector.onPacketSending(NetworkManagerInjector.java:100) ~[ProtocolLib (2).jar:?]
        at com.comphenix.protocol.injector.netty.channel.NettyChannelInjector.processOutbound(NettyChannelInjector.java:566) ~[ProtocolLib (2).jar:?]
        at com.comphenix.protocol.injector.netty.channel.NettyChannelInjector$2.doProxyRunnable(NettyChannelInjector.java:467) ~[ProtocolLib (2).jar:?]
        at com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.proxyRunnable(NettyEventLoopProxy.java:43) ~[ProtocolLib (2).jar:?]
        at com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.execute(NettyEventLoopProxy.java:252) ~[ProtocolLib (2).jar:?]
        at net.minecraft.network.Connection.sendPacket(Connection.java:442) ~[?:?]
        at net.minecraft.network.Connection.send(Connection.java:385) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.send(ServerGamePacketListenerImpl.java:2160) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.send(ServerGamePacketListenerImpl.java:2146) ~[?:?]
        at net.minecraft.server.level.ServerPlayer.openMenu(ServerPlayer.java:1548) ~[?:?]
        at net.minecraft.world.entity.vehicle.ContainerEntity.interactWithContainerVehicle(ContainerEntity.java:93) ~[paper-1.19.4.jar:git-Paper-550]
        at net.minecraft.world.entity.vehicle.AbstractMinecartContainer.interact(AbstractMinecartContainer.java:164) ~[?:?]
        at net.minecraft.world.entity.player.Player.interactOn(Player.java:1128) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl$5.performInteraction(ServerGamePacketListenerImpl.java:2836) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl$5.a(ServerGamePacketListenerImpl.java:2856) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket$InteractionAction.dispatch(ServerboundInteractPacket.java:131) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.dispatch(ServerboundInteractPacket.java:80) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleInteract(ServerGamePacketListenerImpl.java:2797) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:67) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundInteractPacket.handle(ServerboundInteractPacket.java:12) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:51) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.4.jar:git-Paper-550]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1342) ~[paper-1.19.4.jar:git-Paper-550]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[paper-1.19.4.jar:git-Paper-550]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1319) ~[paper-1.19.4.jar:git-Paper-550]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1312) ~[paper-1.19.4.jar:git-Paper-550]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1290) ~[paper-1.19.4.jar:git-Paper-550]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1178) ~[paper-1.19.4.jar:git-Paper-550]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper-1.19.4.jar:git-Paper-550]
        at java.lang.Thread.run(Thread.java:840) ~[?:?]
Caused by: com.google.gson.JsonSyntaxException: Expected id to be a string, was an array ([140...94])
        at net.minecraft.util.GsonHelper.convertToString(GsonHelper.java:85) ~[?:?]
        at net.minecraft.util.GsonHelper.getAsString(GsonHelper.java:91) ~[?:?]
        at net.minecraft.network.chat.HoverEvent$EntityTooltipInfo.create(HoverEvent.java:177) ~[?:?]
        at net.minecraft.network.chat.HoverEvent$Action.deserialize(HoverEvent.java:136) ~[?:?]
        at net.minecraft.network.chat.HoverEvent.deserialize(HoverEvent.java:80) ~[?:?]
        at net.minecraft.network.chat.Style$Serializer.getHoverEvent(Style.java:405) ~[?:?]
        at net.minecraft.network.chat.Style$Serializer.deserialize(Style.java:377) ~[?:?]
        at net.minecraft.network.chat.Style$Serializer.deserialize(Style.java:364) ~[?:?]
        at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:76) ~[gson-2.10.jar:?]
        at com.google.gson.Gson.fromJson(Gson.java:1214) ~[gson-2.10.jar:?]
        at com.google.gson.Gson.fromJson(Gson.java:1319) ~[gson-2.10.jar:?]
        at com.google.gson.Gson.fromJson(Gson.java:1290) ~[gson-2.10.jar:?]
        at com.google.gson.internal.bind.TreeTypeAdapter$GsonContextImpl.deserialize(TreeTypeAdapter.java:179) ~[gson-2.10.jar:?]
        at net.minecraft.network.chat.Component$Serializer.deserialize(Component.java:361) ~[?:?]
        at net.minecraft.network.chat.Component$Serializer.deserialize(Component.java:222) ~[?:?]
        at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:76) ~[gson-2.10.jar:?]
        at com.google.gson.Gson.fromJson(Gson.java:1214) ~[gson-2.10.jar:?]
        at com.google.gson.Gson.fromJson(Gson.java:1319) ~[gson-2.10.jar:?]
        at com.google.gson.Gson.fromJson(Gson.java:1261) ~[gson-2.10.jar:?]
        at net.minecraft.network.chat.Component$Serializer.fromJson(Component.java:536) ~[?:?]
        at lonelibs.net.kyori.adventure.platform.bukkit.MinecraftComponentSerializer.serialize(MinecraftComponentSerializer.java:253) ~[LoneLibs.jar:?]
        ... 42 more
[16:27:18 ERROR]: Parameters: 
  net.minecraft.network.protocol.game.PacketPlayOutOpenWindow@fb15c91[
    a=42
    b=net.minecraft.world.inventory.Containers@58130b15
    c=translation{key='entity.minecraft.hopper_minecart', args=[]}[style={hoverEvent=HoverEvent{action=<action show_entity>, value='net.minecraft.network.chat.ChatHoverable$b@abb7c22e'},insertion=53eb9b77-232e-4aad-b652-c9289afb7082}]
  ]

**Comment by Neast1337** [comment]
I tried to put my server in JAVA 21 but I still get the error, I have everything up to date.

**Comment by LoneDev6** [comment]
Duplicate of: https://github.com/PluginBugs/Issues-ItemsAdder/issues/3282#issuecomment-1875796399
Try to update LoneLibs to the latest version and let me know.

**Comment by Nunchakin** [comment]
Hello is this issue solved? I have the exact same problem, latest ProtocolLib dev build, latest ItemsAdder and LoneLibs:
https://pastebin.com/AyzmcFL4

**Comment by LoneDev6** [comment]
different error.
please next time create a new issue report.
I created one for you, will be fixed in the upcoming update today: #3758

**Comment by Gecenin0fkesi** [comment]
this error still happening on my 1.19.4 server, even tho every dependency is up-to-date.
